src/bundle/Resources/views/themes/admin/content_type/create.html.twig

```diff
@@ -49,15 +49,15 @@
                             {% endblock %}
 
                             {% block left_column %}
-                                {{ form_row(form.name) }}
-                                {{ form_row(form.identifier) }}
-                                {{ form_row(form.description) }}
-                                {{ form_row(form.nameSchema) }}
-                                {{ form_row(form.urlAliasSchema) }}
-                                {{ form_row(form.isContainer) }}
-                                {{ form_row(form.defaultSortField) }}
-                                {{ form_row(form.defaultSortOrder) }}
-                                {{ form_row(form.defaultAlwaysAvailable) }}
+                                {{ form_row(form.name, { row_attr: { class: 'ibexa-content-type-edit__field ibexa-form-field' }}) }}
```

Comment:
nitpick, can we split those into more lines to make it easier to read - here and below?
```suggestion
                                {{ form_row(form.name, { 
                                     row_attr: { 
                                         class: 'ibexa-content-type-edit__field ibexa-form-field',
                                     }})
                                 }}
```